README.md

```diff
@@ -440,3 +456,30 @@ linux-system-roles-12-ddtrs           1/1       Running      57         55d
 ...
 $ oc exec linux-system-roles-12-ddtrs -- cat /var/log/test-harness_linux-system-roles-12-ddtrs.log
 ```
+
+## run-tests Environment Variables
+
+The easiest way to customize the pod is via environment variables.  The
+environment variables correspond to the command line arguments to `run-tests`.
+The environment variables will override the command line parameters, if both
+are used.  In general, the name of the environment variable is the name of the
+command line argument, in all upper case, with `_` instead of `-`, and with
+the prefix `TEST_HARNESS_`.  For example, the command line option
+`--use-images` can be set via the env. var. `TEST_HARNESS_USE_IMAGES`.  See
+`run-tests` usage for the complete list.  In addition, there are some other
+env. vars which are useful for debugging:
+* `TEST_HARNESS_CONFIG_NAME` - replaces `config["name"]` - used for the name in
+the PR status message
+* `TEST_HARNESS_REPOSITORIES` - comma delimited list of repos to override
+`config["repositories"]`
+* `TEST_HARNESS_KEEP_RESULTS` - if `true`, this will cause the test not to
+delete the temp directory where it holds the results before uploading them
+
+For example, in a running OpenShift cluster, if you wanted to use a custom name for the PR status:
```

Comment:
The lines seem to be wrapped inconsistently.

Reply:
https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0